src/components/networks/utils.js

```diff
@@ -53,6 +53,12 @@ export function ipv4IsNetworkIdentifier(address, netmask) {
     return address === ipaddr.IPv4.networkAddressFromCIDR(`${address}/${prefix}`).toString();
 }
 
+export function ipv4ExampleBridgeAddressForNetworkIdentifier(network) {
+    const net = ipaddr.parse(network);
+    net.octets[3] += 1;
```

Comment:
Is that always right? AFAICS, `ipv4IsNetworkIdentifier()` would also return true if you specified e.g. a netmask of /32? And if the last octet is 255, then this would give an invalid address.

Reply:
Yeah, that would break, that's why I called this ExampleBridgeAddress. If we use this to produce an address that gets stored in configuration, it would need to be a bit more careful, I guess. 

A prefix length of 32 is not a network, it's a single address, and we could reject it.

Reply:
Ok, if we allow all addresses for a /32 "subnet", the next check will fail "IPv4 address cannot be same as the network's broadcast address". The same happens for /31: this is a subnet with only two "addresses": one is the network id, the other is the broadcast address. So that is out as well. Let's just reject /32 and /31 subnets.

Reply:
Also, we include `localPtr="yes"` in our network XML definition, and that requires a prefix <= 30. So that's another reason to reject prefix >= 31.

Reply:
Alright localPtr="yes" actually has more restrictions: IPv4 prefix must be divisible by 8.  It it's not, trying to activate the network will fail and actually disconnect Cockpit.